index.js

```diff
@@ -155,13 +155,5 @@ module.exports = {
   },
   included: function(app) {
     this._super.included(app);
-
```

Comment:
Correct.  The versioning can now be controlled by what ever version the consuming app has in the package.json.  There is no longer the need to pick a version from the browserified distribution of yadda.